crates/db_schema/src/impls/community.rs

```diff
@@ -332,12 +332,22 @@ impl ApubActor for Community {
     )
   }
 
-  fn read_from_name(conn: &PgConnection, community_name: &str) -> Result<Community, Error> {
+  fn read_from_name(
+    conn: &PgConnection,
+    community_name: &str,
+    include_deleted: bool,
+  ) -> Result<Community, Error> {
     use crate::schema::community::dsl::*;
-    community
+    let q = community
       .filter(local.eq(true))
-      .filter(lower(name).eq(lower(community_name)))
-      .first::<Self>(conn)
+      .filter(lower(name).eq(lower(community_name)));
+    if include_deleted {
+      q.first::<Self>(conn)
+    } else {
+      q.filter(deleted.eq(false))
+        .filter(removed.eq(false))
+        .first::<Self>(conn)
+    }
```

Comment:
This is a little awkward, not sure if theres a better way to write the query.

Reply:
Like this? Gives me error "overflow evaluating the requirement `_: Sized`".
```

    let mut q = community
      .filter(local.eq(true))
      .filter(lower(name).eq(lower(community_name))).into_boxed();
    if !include_deleted {
      q = q.filter(deleted.eq(false))
        .filter(removed.eq(false)).into_boxed();
    }
    q.first::<Self>(conn)
```

Reply:
You do the into_boxed before filters I think.